subprojects/flork/src/thread/spinlock.zig

```diff
@@ -31,7 +26,8 @@ pub const Spinlock = struct {
 
     /// Release lock without restoring interrupt state
     pub fn ungrab(self: *@This()) void {
-        _ = @atomicRmw(usize, &self.serving, .Add, 1, .AcqRel);
+        const serving = @atomicLoad(usize, &self.serving, .Monotonic);
+        @atomicStore(usize, &self.serving, serving + 1, .Release);
```

Comment:
Isn't this just `@atomicStore(usize, &self.serving, self.serving + 1, .Release);`